crypto/src/bls12381.rs

```diff
@@ -368,6 +392,19 @@ impl From<BLS12381PrivateKey> for BLS12381KeyPair {
     }
 }
 
+impl EncodeDecodeBase64 for BLS12381KeyPair {
+    fn decode_base64(value: &str) -> Result<Self, eyre::Report> {
+        keypair_decode_base64(value)
+    }
+
+    fn encode_base64(&self) -> String {
+        let mut bytes: Vec<u8> = Vec::new();
+        bytes.extend_from_slice(self.secret.as_ref());
```

Comment:
Unfortunately, we have to copy for now, perhaps we can use some method that allows us to borrow the PK within the crypto crate.